content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/integrating_landed_costs.rst

```diff
@@ -73,8 +73,8 @@ When creating new vendor bills, this product can be added as an invoice line as
 
 .. important::
    To apply a landed cost on a vendor bill, the products included in the original |PO| **must**
-   belong to a *Product Category* with its *Force Removal Strategy* set to |FIFO|, and its *Costing
-   Method* set to |AVCO|.
+   belong to a *Product Category* with its *Force Removal Strategy* set to |FIFO|. The *Costing
+   Method* can be set to either |AVCO| or |FIFO|.
```

Comment:
Good question. Personally, I also lean on the side of providing *any* kind of information like this, without assuming that it'll already be common knowledge. It only ensures that the proper info is being delivered, and since it's a quick mention, it doesn't get in the way of anything. So, that said, I'd suggest mentioning that, just in case the reader might not be aware of it. That's my "two cents" on the matter, anyway... 👍